python/pyspark/sql/tests/test_session.py

```diff
@@ -39,7 +39,9 @@ class SparkSessionTests1(ReusedSQLTestCase):
     def test_sparksession_with_stopped_sparkcontext(self):
         self.sc.stop()
         sc = SparkContext("local[4]", self.sc.appName)
-        spark = SparkSession.builder.getOrCreate()
+        spark = SparkSession.getActiveSession()
```

Comment:
I think for test case, it's fine to don't fix.

Reply:
Makes sense, restored.